src/util/v4/v4_portable_v1.h

```diff
@@ -60,17 +60,26 @@ namespace v4
 
     friend inline v4    czero( const v4int &c, const v4 &a ) ALWAYS_INLINE;
     friend inline v4 notczero( const v4int &c, const v4 &a ) ALWAYS_INLINE;
-    friend inline v4 merge( const v4int &c, const v4 &a, const v4 &b ) ALWAYS_INLINE;
+    friend inline v4    merge( const v4int &c, const v4 &a, const v4 &b ) ALWAYS_INLINE;
 
     // v4 memory manipulation friends
 
-    friend inline void   load_4x1( const void * ALIGNED(16) p, v4 &a ) ALWAYS_INLINE;
```

Comment:
Do you think it's worth trying to clean up a bunch of the shared code between `v4_portable_v0` and `v4_portable_v1`?

Reply:
v4_portable_v0 and v4_portable_v1 are really two separate portable implementations.  If you do a visual diff, you will see that the v1 version does a portable implementation with a for loop with a known trip count equal to the vector length.  The v0 version is the original style that Kevin used where you have essentially fully unrolled that for loop.  I originally created the v1 implementation to explore whether the compiler could be smart enough to create assembly code as efficient or better than the intrinsics implementations.  It is really curious that when building a portable version of VPIC, depending on the vector length, compiler and architecture, one version is more performant than the other in unpredictable ways.  I want to retain both versions for now for possible future use and studies.